pkg/notifications/types.go

```diff
@@ -742,7 +820,9 @@ type AiNotificationsDestinationInput struct {
 	// name
 	Name string `json:"name"`
 	// properties
-	Properties []AiNotificationsPropertyInput `json:"properties,omitempty"`
+	Properties []AiNotificationsPropertyInput `json:"properties"`
```

Comment:
So this is needed, as the type of this on NG is `[AiNotificationsPropertyInput!]!`, it simply has never been tested with an empty value because there is always something there.